core/build.gradle.kts

```diff
@@ -6,11 +6,11 @@ plugins {
   id("com.palantir.docker") version "0.26.0"
   id("com.palantir.docker-run") version "0.25.0"
   id("org.jetbrains.dokka") version "0.10.1"
-  id("com.jfrog.bintray") version "1.8.5"
   `maven-publish`
+  signing
 }
 
-group = "com.kotlin.samples.verifier"
+group = "io.github.alexanderprendota"
```

Comment:
the name should be `com.kotlin.samples.verifier`

Reply:
Unfortunately according to the rules https://central.sonatype.org/publish/requirements/coordinates/  
>... This means that if you are the owner or maintainer of a domain name 

we should confirm to own kotlin.com

Reply:
oh, got it. But maybe we have a way not to publish with "alexanderprendota" suffix. like "io.site.snippets.verifier" or something else